spec/lib/hyrax/configuration_spec.rb

```diff
@@ -16,55 +16,56 @@
     it { is_expected.to eq('default') }
   end
 
-  it { is_expected.to respond_to(:active_deposit_agreement_acceptance?) }
   it { is_expected.to respond_to(:active_deposit_agreement_acceptance=) }
+  it { is_expected.to respond_to(:active_deposit_agreement_acceptance?) }
   it { is_expected.to respond_to(:activity_to_show_default_seconds_since_now) }
   it { is_expected.to respond_to(:admin_set_predicate) }
   it { is_expected.to respond_to(:admin_set_predicate=) }
   it { is_expected.to respond_to(:analytic_start_date) }
   it { is_expected.to respond_to(:analytics?) }
-  it { is_expected.to respond_to(:analytics) }
-  it { is_expected.to respond_to(:arkivo_api?) }
+  it { is_expected.to respond_to(:analytics_provider) }
+  it { is_expected.to respond_to(:analytics_provider=) }
```

Comment:
this is the important part.  The rest of the changes in this file are just 1 man's love of the alphabetical order.